### DISCUSSION TITLE
TP2 Ejercicio 7. Funcion de Potencia

### DISCUSSION THREAD

**Discussion by Elias-Turner** [INGCOM-UNRN-P1/cursada-2023#79] [Preguntas y respuestas] [answered]
Contexto: Hice una función que haga una potencia normal, es decir, con `*`. Para darme un número determinado. después hice una función que usando un `while` que dice `numero<resultadoP` (resultado del número elevado a la potencia) haga `numero+=numero`

Mi pregunta es la siguiente: ¿es válido esta forma?

**Comment by martinvilu** [comment, marked as answer]
> Desarrolla una función que calcule la potencia de un número elevado a una potencia dada, sin utilizar el operador de multiplicación.

Estarías usando el operador de multiplicación de manera indirecta; ahora; si haces función que sea la multiplicación con sumas, cumpliría la consigna.
